src/module.cc

```diff
@@ -46,7 +46,7 @@ bool Module::matchesRequestPath(const std::string& reqPath) const {
         pathSize--;
     }
 
-    for (int i = 0; i < pathSize; ++i) {
+    for (size_t i = 0; i < pathSize; ++i) {
```

Comment:
haha it was because I was comparing a signed and unsigned! not sure why it wasn't an error before...